server/data/index.ts

```diff
@@ -11,12 +11,10 @@ initialiseAppInsights()
 buildAppInsightsClient(applicationInfo)
 
 // eslint-disable-next-line import/order
-import { AuthenticationClient } from '@ministryofjustice/hmpps-auth-clients'
```

Comment:
This is the essence of the change .... import `RedisTokenStore` and `InMemoryTokenStore` from `hmpps-auth-clients` rather than our own code. 
Ultimately that is all I've really done in this PR.
Everything else in this PR is about adjusting the imports for tests etc, and deleting the versions of these classes from LWP's codebase 👍